apps/docs/layouts/MainSkeleton.tsx

```diff
@@ -359,30 +322,41 @@ const NavContainer = memo(function NavContainer({ menuId }: { menuId: MenuId })
 })
 
 function MainSkeleton({ children, menuId }: PropsWithChildren<{ menuId: MenuId }>) {
+  const mobileMenuOpen = useMenuMobileOpen()
+  const isHomePage = menuId === 'home'
+
   return (
-    <div className="flex flex-row h-full">
-      <NavContainer menuId={menuId} />
-      <Container className="[--header-height:40px] lg:[--header-height:60px]">
-        <div className={['lg:sticky top-0 z-10 overflow-hidden'].join(' ')}>
-          <TopNavBar />
-        </div>
-        <div
-          className={[
-            'sticky transition-all top-0',
-            'z-10',
-            'backdrop-blur backdrop-filter bg-background',
-          ].join(' ')}
-        >
-          <div className={['lg:hidden', 'px-3.5', 'border-b z-10'].join(' ')}>
-            <MobileHeader menuId={menuId} />
+    <div className="flex flex-col h-full w-full">
+      <div className="hidden lg:sticky w-full lg:flex top-0 left-0 right-0 z-50">
+        <TopNavBar />
+      </div>
+      <div className="flex flex-row h-full">
+        {!isHomePage && <NavContainer menuId={menuId} />}
+        <Container className="[--header-height:40px] lg:[--header-height:90px]">
```

Comment:
Custom properties should be moved back up so they can apply to the nav bar as well, since they should be used to control the height (this syncs the header height with the table of contents positioning and possibly scroll margin)

Reply:
Could also extend tailwind config for Docs so you've got more tailwind classes available with custom values ?

`[--header-height:40px]` this is also a nice approach though, assuming it sets a css variable.